MAUI/Picker/dealing-with-column-header.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title: Column header with .NET MAUI Picker control | Syncfusion
+description: Learn here all about column header with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Dealing with Column Header in .NET MAUI Picker (SfPicker)
+
+This section explains about the column header customization of picker control.
```

Comment:
This section explains the column header customization of picker control.

Reply:
Content modified